src/main/java/werkit/UI.java

```diff
@@ -224,15 +224,20 @@ public String getColorText(String color, String text) {
      * Prints the help messages. To be updated.
      */
     public void printHelpMessage() {
-        printListHelp();
+        printExerciseHelp();
         printLine();
-        printExerciseListHelp();
         printLine();
-        printWorkoutAddHelp();
+        printWorkoutHelp();
         printLine();
-        printWorkoutDeleteHelp();
         printLine();
-        printWorkoutUpdateHelp();
+        printPlanHelp();
+        printLine();
+        printLine();
+        printScheduleHelp();
+        printLine();
+        printLine();
+        printSearchHelp();
+        printLine();
```

Comment:
Nice job separating and generalizing the methods!

Reply:
Thank you!